src/controllers/profile.controller.ts

```diff
@@ -8,3 +10,69 @@ export const getProfile = async (req: Request, res: Response) => {
     res.status(500).json({ error: err });
   }
 };
+
+export const updateProfile = async (req: Request, res: Response) => {
+  try {
+    const user: any = req.user;
+    const {
+      primary_email,
+      contact_email,
+      first_name,
+      last_name,
+      image_url,
+      linkedin_url,
+    }: any = req.body;
```

Comment:
The type should be Profile here,

```suggestion
    }: Profile = req.body;
```